src/mail-app/mailLocator.ts

```diff
@@ -710,6 +710,7 @@ class MailLocator {
 				calendarFacade,
 				this.entityClient,
 				this.logins,
+				(isBrowser() || isDesktop()) ? PushIdentifierAppType.Integrated : PushIdentifierAppType.Calendar, //TODO: Each locator will have a nativeInterface and must provide this. Desktop, browser and mail (temporally) should use integrated.
```

Comment:
we have separate locators now, is it already possible?

Reply:
yes, this is already resolved